airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicationActivity.java

```diff
@@ -18,6 +18,7 @@ public interface ReplicationActivity {
   StandardSyncOutput replicate(JobRunConfig jobRunConfig,
                                IntegrationLauncherConfig sourceLauncherConfig,
                                IntegrationLauncherConfig destinationLauncherConfig,
-                               StandardSyncInput syncInput);
+                               StandardSyncInput syncInput,
+                               final String taskQueue);
```

Comment:
Talked about this on zoom, but to recap in writing:

Because this is changing the signature of an ActivityMethod, we should be careful around how this would affect any running workflows when we deploy the change.

I _think_ that Temporal will basically replay the activity by passing in 'null' for any existing workflows that started before this was added, so we should make sure the code can handle 'null' as an input. Might be good to add a `@Nullable` annotation here with a comment explaining that this is to handle backwards compatibility with older Temporal workflows.

Once we're confident the older workflows are all finished, we can remove the @Nullable annotation and cleanup if we'd like.